FCCee/IDEA/compact/IDEA_o1_v02/detailedMuonSystem_v00.xml

```diff
@@ -0,0 +1,210 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lccdd>
+
+  <define>
+
+     <!---                            Muon System Parameters                       -->
+
+     <!-- %%%%%%                      microRWELL chamber different layers thicknesses                %%%%%% -->
+
+       <constant name = "G10_FR4Thick"                          value = "1.6*mm"/>
+       <constant name = "CuThick"                               value = "0.035*mm"/>
+       <constant name = "GasLayerThick"                         value = "6*mm"/>
+       <constant name = "Cu2Thick"                              value = "0.005*mm"/>
+       <constant name = "KaptonThick"                           value = "0.05*mm"/>
+       <constant name = "CarnonFiberThick"                      value = "0.0001*mm"/>
+       <constant name = "CarbonFiber2Thick"                     value = "0.1*mm"/>
+       <constant name = "SiThick"                               value = "1.6*mm"/>
+       <constant name = "mRWELLTotalThickness"                  value = "G10_FR4Thick+CuThick+GasLayerThick+Cu2Thick+KaptonThick+CarnonFiberThick+CuThick+CarbonFiber2Thick+CuThick+SiThick"/> <!-- This sequense is the current order of mRWELL (total 10 layers) -->
+
+       <constant name = "mRWELLYLength"                              value = "250*mm"/>
+       <constant name = "mRWELLZLength"                              value = "250*mm"/>
+
+       <constant name = "microRWELLVolumeThick"                      value = "60*mm"/> <!--The thickness of the detector layer including the microRWELL material and the spacing between the layers too -->
+       <constant name = "microRWELLVolumeThickEndcap"                value = "120*mm"/>
+
+       <constant name = "overlapingY"                                   value = "10*mm"/> <!-- the common distance between mRWELL chambers in Y direction -->
+       <constant name = "overlapingZ"                                   value = "10*mm"/> <!-- the common distance between mRWELL chambers in Z direction -->
+
+     <!-- %%%%%%                      Iron yoke thickness                %%%%%% -->
+     
+      <constant name = "YokeThickness"                               value = "300*mm"/>  
+
+
+
+  </define>   
+
+
+  <display>
+     <vis name="yoke_vis"  r="255/256" g="0/256"   b="0/256"   alpha="1.0"  showDaughters="true" visible="true" />
+     <vis name="no_vis" showDaughters="true" visible="false" />
+  </display>
+
+  <readouts>
+     <readout name="mRWELLChamberReadout">
+     <segmentation type="CartesianGridXY" grid_size_x="1*cm" grid_size_y="1*cm"/>   <!-- Depending on cell size 1cm*1cm  -->
+     <id>system:4,chamber:9,layer:4,side:4,x:6,y:-6</id>
+    </readout>
+  </readouts>   
+
+
+
+  <detectors>
+
+    <!-- mRWELL envelope -->
+    <detector name="muonSystem1" type="muonSystem_o1_v00"  vis="MuonVisEnv" id="51" readout="mRWELLChamberReadout">
```

Comment:
Here and in other places: the "id" should be defined in `DectDimensions_IDEA_o1_v01.xml` and called here (see and modify https://github.com/key4hep/k4geo/blob/a02cbb463f412055d0c788f58b799e0b95cc8aae/FCCee/IDEA/compact/IDEA_o1_v02/DectDimensions_IDEA_o1_v01.xml#L43)

Reply:
I also suspect this should crash as it is now (did you try running it with ddsim?) because we have 4 bits for `system` in the readout which only allows to have "id's" from 0 to 15.

Reply:
I added the 9 muon system layers IDs to the DectDimensions_IDEA_o1_v01.xml, and ddsim run without crashes even if the system has 4 bits. Isn't the 4 bits in that case only for the muon system, since for example I have here 9 detectors (layers)? since also every sub-detector has its own readout description?